examples/plot_positions.py

```diff
@@ -12,63 +12,103 @@
 from eeg_positions import get_elec_coords, plot_coords
 
 # %%
-# Get the electrode positions!
-# Let's start with the basic 10-20 system in 2 dimensions (``"x"`` and ``"y"``).
+# Let's start with the basic 10-20 system in two dimensions:
 
 coords = get_elec_coords(
     system="1020",
     dim="2d",
 )
 
-# `coords` is a pandas.DataFrame object
+# %%
+# This function returns a ``pandas.DataFrame`` object:
+
 coords.head()
 
 # %%
 # Now let's plot these coordinates.
 # We can supply some style arguments to :func:`eeg_positions.plot_coords` to control
-# the color of the scatter dots, and the text annotations.
+# the color of the electrodes and the text annotations.
 
 fig, ax = plot_coords(
-    coords, scatter_kwargs=dict(color="green"), text_kwargs=dict(fontsize=10)
+    coords, scatter_kwargs={"color": "g"}, text_kwargs={"fontsize": 10}
 )
 
-ax.axis("off")
 fig
 
 # %%
-# Notice in the above plot, that the "landmarks" are there: ``NAS``, ``LPA``,
-# and ``RPA``. We can drop these by passing the ``drop_landmarks=True`` to
-# :func:`get_elec_coords`.
+# Notice that the "landmarks" ``NAS``, ``LPA``, and ``RPA`` are included. We can drop
+# these by passing ``drop_landmarks=True`` to :func:`get_elec_coords`:
 
 coords = get_elec_coords(
     system="1020",
     drop_landmarks=True,
     dim="2d",
 )
 
-
 fig, ax = plot_coords(
-    coords, scatter_kwargs=dict(color="green"), text_kwargs=dict(fontsize=10)
+    coords, scatter_kwargs={"color": "g"}, text_kwargs={"fontsize": 10}
 )
 
-ax.axis("off")
 fig
 
 # %%
-# We can also plot in 3D. Let's pick a system with more electrodes now.
+# Often, we might have a list of electrode names that we would like to plot. For
+# example, let's assume we have the following 64 channel labels (based on the 10-05
+# system):
+
+chans = """Fp1 AF7 AF3 F1 F3 F5 F7 Fp2 AF8 AF4 F2 F4 F6 F8 FT7 FC5 FC3
+FC1 C1 C3 C5 T7 TP7 CP5 CP3 CP1 FT8 FC6 FC4 FC2 C2 C4 C6 T8 TP8 CP6 CP4
+CP2 P1 P3 P5 P7 P9 PO7 PO3 O1 P2 P4 P6 P8 P10 PO8 PO4 O2 Iz Oz POz Pz
+Fz AFz Fpz CPz Cz FCz""".split()
+
+# %%
+# Many experiments aggregate electrodes into regions of interest (ROIs), which we could
+# visualize with different colors. Let's get their coordinates first:
+
+coords = get_elec_coords(elec_names=chans)
+
+# %%
+# Now we specifiy individual colors using the ``scatter_kwargs``` argument. We create a
+# list of 64 colors corresponding to our 64 coordinates (in the original order as
+# provided by ``chans``):
+
+colors = (
+    ["salmon"] * 14
+    + ["skyblue"] * 24
+    + ["violet"] * 16
+    + ["lightgreen"] * 7
+    + ["khaki"] * 3
+)
+
+fig, ax = plot_coords(
```

Comment:
to make this the gallery thumbnail (https://sphinx-gallery.github.io/stable/auto_examples/plot_4_choose_thumbnail.html#sphx-glr-auto-examples-plot-4-choose-thumbnail-py)

```suggestion
# sphinx_gallery_thumbnail_number = 3
fig, ax = plot_coords(
```